prime-router/src/main/kotlin/history/azure/DatabaseDeliveryAccess.kt

```diff
@@ -41,8 +42,12 @@ class DatabaseDeliveryAccess(
         organization: String,
         orgService: String?,
     ): Condition {
-        var filter = ACTION.ACTION_NAME.eq(TaskAction.batch)
-            .and(REPORT_FILE.RECEIVING_ORG.eq(organization))
+        var filter = (
```

Comment:
Can you add a comment explaining why need this - " This is a temporary fix to make the daily data page show data for reports routed through the ELR ELIMS topic since reports routed through the ELR-ELIMS topic skip the batch step. There are plans to have a dashboard for receivers that works for all messages routed through RS. "